### PR TITLE
Price Floors: Update Cache-Control header toleration

### PR DESCRIPTION
**Pull request by SerhiiNahornyi**

### CODE REVIEW

File: src/main/java/org/prebid/server/floors/PriceFloorFetcher.java [hunk 1]
```diff
@@ -197,9 +202,12 @@ private static int resolveMaxRules(Long accountMaxRules) {
 
     private Long cacheTtlFromResponse(HttpClientResponse httpClientResponse, String fetchUrl) {
         final String cacheMaxAge = httpClientResponse.getHeaders().get(HttpHeaders.CACHE_CONTROL);
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/floors/PriceFloorFetcher.java:204]: Bad variable name. It's not cache max age, it's cache-control header value.

File: src/main/java/org/prebid/server/floors/PriceFloorFetcher.java [hunk 2]
```diff
@@ -39,13 +39,18 @@
...
 
 public class PriceFloorFetcher {
 
     private static final Logger logger = LoggerFactory.getLogger(PriceFloorFetcher.class);
 
     private static final int ACCOUNT_FETCH_TIMEOUT_MS = 5000;
     private static final int MAXIMUM_CACHE_SIZE = 300;
+    private static final int MIN_MAX_AGE_SEC_VALUE = 600;
+    private static final int MAX_AGE_SEC_VALUE = Integer.MAX_VALUE;
+    private static final Pattern CACHE_CONTROL_HEADER_PATTERN = Pattern.compile("^.*(max-age=\\d+).*$");
```
Comment (Net-burst) [inline src/main/java/org/prebid/server/floors/PriceFloorFetcher.java:53]: You're only interested in the value of the `max-age`. Move your `/d+` matcher to the capture group and access it directly. You won't need to split your result then.

Comment (SerhiiNahornyi) [reply src/main/java/org/prebid/server/floors/PriceFloorFetcher.java]: Nice catch, thx)

File: src/main/java/org/prebid/server/floors/PriceFloorFetcher.java (hunk 1 above)
Comment (SerhiiNahornyi) [reply src/main/java/org/prebid/server/floors/PriceFloorFetcher.java]: Done

**Review by Net-burst — APPROVED** [review]
LGTM

**Pull request merged**